generators/server/index.js

```diff
@@ -583,8 +583,8 @@ module.exports = class JHipsterServerGenerator extends BaseBlueprintGenerator {
           'java:war:prod': 'npm run java:war -- -Pprod',
           'java:docker:dev': 'npm run java:docker -- -Pdev,webapp',
           'java:docker:prod': 'npm run java:docker -- -Pprod',
-          'ci:backend:test': 'npm run backend:info && npm run backend:doc:test && npm run backend:nohttp:test && npm run backend:unit:test',
-          'ci:server:package': 'npm run java:$npm_package_config_packaging:$npm_package_config_default_environment',
+          'ci:backend:test':
+            'npm run backend:info && npm run backend:doc:test && npm run backend:nohttp:test && npm run backend:unit:test -- -P$npm_package_config_default_environment',
```

Comment:
Environment will be passed to the last command (test:unit) only.
Do we need to pass to others commands (info, nohttp) too?

Reply:
I don't think so (the previous steps are only to generate doc), but if someone can confirm I'll sleep a little bit better.
I'll test on my side if I get no answer

Reply:
Tested on my side: It's ok for the doc and info without setting the maven profile: there's no change in the output at all